github/util.py

```diff
@@ -221,3 +222,82 @@ def retrieve_email(username: str):
         )
     )
 
+def _create_team(
+    github: GitHub,
+    organization_name: str,
+    team_name: str
+):
+    # passed GitHub object must have org. admin authorization to create a team
+    organization = github.organization(organization_name)
+    team_list = list(filter(lambda t: t.name == team_name, organization.teams()))
+    if team_list:
+        util.verbose("Team {name} already exist".format(name=team_name))
+        return
+
+    try:
+        organization.create_team(name=team_name)
+        util.info("Team {name} created".format(name=team_name))
+    except ForbiddenError as err:
+        util.fail("{msg} Cannot create team {name} in org {org} due to missing privileges".format(
+            msg=err,
+            name=team_name,
+            org=organization_name
+        ))
+
+def _add_user_to_team(
+    github: GitHub,
+    organization_name: str,
+    team_name: str,
+    user_name: str
+):
+    # passed GitHub object must have org. admin authorization to add a user to a team
+    organization = github.organization(organization_name)
+    team_list = list(filter(lambda t: t.name == team_name, organization.teams()))
+    if not team_list:
+        util.fail("Team {name} does not exist".format(name=team_name))
+
+    team = team_list[0]
+    if team.is_member(user_name):
+        util.verbose("{username} is already assigned to team {teamname}".format(
+            username=user_name,
+            teamname=team_name
+        ))
+        return
+
+    if team.add_member(username=user_name):
+        util.info("Added {username} to team {teamname}".format(
+            username=user_name,
+            teamname=team_name
+        ))
+    else:
+        util.fail("Could not add {username} to team {teamname}. Please check for missing privileges".format(
+            username=user_name,
+            teamname=team_name
+        ))
+
+def _add_all_repos_to_team(
+    github: GitHub,
+    organization_name: str,
+    team_name: str,
+    permission: str='admin'
+):
+    # passed GitHub object must have org. admin authorization to assign team to repo with admin rights
+    organization = github.organization(organization_name)
+    team_list = list(filter(lambda t: t.name == team_name, organization.teams()))
+    if not team_list:
+        util.fail("Team {name} does not exist".format(name=team_name))
+
+    team = team_list[0]
+    for repo in organization.repositories():
+        if team.has_repository(repo.full_name):
+            util.verbose("Team {teamnname} already assigned to repo {reponame}".format(
+                teamnname=team_name,
+                reponame=repo.full_name
+            ))
+            continue
+
+        team.add_repository(repository=repo.full_name, permission=permission)
```

Comment:
does thy github3 api not define the allowed permission values? :-(

Reply:
Done